### ISSUE TITLE
Scaling on beta makes cards look pixelated

### ISSUE THREAD

**Issue by mainman879** [Cockatrice/Cockatrice#4643]
**System Information:**

Client Version: 2.8.1-beta.9 (2022-05-12)
Client Operating System: Windows 10 Version 2009
Build Architecture: 64-bit
Qt Version: 6.3.0
System Locale: en_US
Install Mode: Standard
_______________________________________________________________________________________

Most computers have their displays set to 100% scaling by default, and on these settings the cards look [fine](https://i.gyazo.com/45a9c972df84c23bf4ec002f90011eac.png). But when scale and layout is set to 125% or higher in display settings, the cards look pixelated [like so](https://i.gyazo.com/f3e971c68a70c2359c8e917ee92e37eb.png).

_______________________________________________________________________________________

**Steps to reproduce:**
 - Use 100% scaling on a windows monitor
 - Look at cards
 - Use 125% scaling instead and look at the difference

Side note: This seems to be a beta only bug (could be related to qt changes), some further testing may be needed.

**Comment by tooomm** [comment]
I can confirm this behavior on Windows with `2.8.1-beta.9` and `Qt 6.3.0`
It's no issue on the last stable release:
```
Client Version: 2.8.0 (2021-01-26)
Client Operating System: Windows 10 (10.0)
Build Architecture: 64-bit
Qt Version: 5.12.9
System Locale: en_GB
Install Mode: Standard
```

**Comment by Elarnon** [comment]
Fixed by #4962
